code/modules/mob/living/living_defines.dm

```diff
@@ -50,6 +50,8 @@
 
 	var/list/say_log = list() //a log of what we've said, plain text, no spans or junk, essentially just each individual "message"
 
+	var/bubble_icon = "default" //what icon the mob uses for speechbubbles

```

Comment:
isn't the point of a feature freeze to minimize any potentially bug causing additions though
I mean i have no problem with this feature but i just don't think its in the spirit of the freeze
freezes are stupid btw